content/english/useless-box/appendix.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Build the box"
+description: ""
+draft: true
+weight: 3
```

Comment:
All weight: values should go in the order of the pages. So this should be 5, for example